### PR TITLE
[FR] Update Beatmap Editor/Design

### PR DESCRIPTION
**Pull request by DoryanR**
Translation of the Design page

### CODE REVIEW

**Comment by MegaApplePi** [comment]
@DoryanR STOP I am trying to merge your pull requests in now, fixing the conflicts has been fixed on my end

**Comment by DoryanR** [comment]
Oh, sorry, I thought I was supposed to solve them on my own.

**Comment by DoryanR** [comment]
@MegaApplePi If everything is good for you, you can merge, sorry for this mistake on my part, I thought I had to solve the conflicts by myself.

File: wiki/Beatmap_Editor/Design/fr.md [hunk 1]
```diff
@@ -1,30 +1,160 @@
...
+![Extra commands](img/SE_AO.jpg "Commandes supplémentaires")
+
+| Commande | Utilisation |
+| ------- | ----- |
+| Tweening | Vous avez deux Keyframes (ie Start/End), voulez-vous animer entre elles ? Si oui, allumez Tweening. Sinon, désactivez-le. |
+| Entrée/sortie de l'effacement | Démarrage lent / rapide puis rapide / lent près du point final. Utile pour le fondu enchaîné. |
+| Origine | Point d'origine. Coin "Haut-Gauche" ou "Centre" de l'image |
+| Diff. Spécifique | Sauvegardez tous les SB dans le fichier `.osu` de cette difficulté au lieu du fichier `.osb`. (Voir note ci-dessous) |
+
+
```
Comment (MegaApplePi) [inline wiki/Beatmap_Editor/Design/fr.md]: delete this line

File: wiki/Beatmap_Editor/Design/fr.md [hunk 2]
```diff
@@ -1,30 +1,160 @@
...
-  - A partir de quel point se font les effets (Rotation, Redimensionnement, etc...). Ils peuvent avoir lieu sur: L'angle en haut à gauche ou le Centre de l'image.
-- Possibilité de définir la SB en fonction de la difficulté (Bouton "Diff. specific")
-- Possibilité d'ajouter un Fond d'écran et mettre une couleur de fond.
-
-## Limites
-
-- Pas d'effets sonores, pas très gênant du fait qu'ils pourraient gêner les joueurs si ils sont placés près des [Hit objects](/wiki/Hit_Objects). L'utilisation d'effets sonores n'est autorisée qu'aux mappers expérimentés, avec l'aide d'un BAT.
-- Loops et déclencheurs impossibles à créer.
-- Pas de couleurs ou de commandes de déplacement Move-X/Move-Y.
+# Conception
```
Comment (MegaApplePi) [inline wiki/Beatmap_Editor/Design/fr.md]: You disabled my ability to add commits to this pull request, so I'll have you add it instead.
```suggestion
---
no_native_review: true
---
# Conception
```

**Pull request merged**